internal/pipeline/resolver/config.go

```diff
@@ -2,28 +2,19 @@ package resolver
 
 import (
 	"context"
-	"fmt"
 
 	"github.com/buildkite/cli/v3/internal/config"
 	"github.com/buildkite/cli/v3/internal/pipeline"
 )
 
-func ResolveFromConfig(c *config.Config) PipelineResolverFn {
+func ResolveFromConfig(conf *config.Config) PipelineResolverFn {
 	return func(context.Context) (*pipeline.Pipeline, error) {
-
-		if c == nil {
```

Comment:
We have a validator that will ensure config is not nil at a higher level than this so I opted to remove it